### PR TITLE
Spiritbinding

### PR DESCRIPTION
**Pull request by zumbak04**
## Changelog:
- The mogu can use Build New Vessel if they are too old, have maims, negative genetic traits to create a new body. You will be rejuvenated and lose all the body-related traits.

# How to test:
Start as a mogul, `add_trait weak` to see the decision.

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+EVTDESC_WCRAC_500;You order the most experienced carver you have to sculpt your new body. Together with him, you head to career near the capital. Masons who work there assure that you will not find better stone in the whole country. They tell that there are two types of rock: soft and hard. Your carver responds: it's easier to sculpt soft stone, but a body made of hard one will last you longer and better.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3044]:
```suggestion
EVTDESC_WCRAC_500;You order the most experienced carver you have to sculpt your new body. Together with him, you head to career near the capital. Masons who work there assure you that you will not find better stone in the whole country. They inform you that there are two types of rock: soft and hard. Your carver states that it is easier to sculpt soft stone, but a body made of hard one will last longer and may be of better quality.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+EVTOPTA_WCRAC_500;Soft, I don't want to risk.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3045]:
```suggestion
EVTOPTA_WCRAC_500;Soft, I don't want to take risks.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 3]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+EVTOPTB_WCRAC_500;A hard will in a hard body.;;;;;;;;;;;;;x
+spiritbinding_deformed_body_tooltip;Your new body is deformed.;;;;;;;;;;;;;x
+spiritbinding_excellent_body_tooltip;Your new body is of excellent quality.;;;;;;;;;;;;;x
+EVTOPTA_WCRAC_505_normal;Call the spiritbinders.;;;;;;;;;;;;;x
+EVTOPTA_WCRAC_505_fail;Fine, you may live! Call the spiritbinders.;;;;;;;;;;;;;x
+EVTOPTB_WCRAC_505_normal;My soul isn't ready yet.;;;;;;;;;;;;;x
+EVTOPTB_WCRAC_505_fail;Execute him. This body doesn't suit me.;;;;;;;;;;;;;x
+EVTDESC_WCRAC_505;Finally, the carver presents your new body. You slowly walk around the sculpture. [Root.GetBodyQualityResponse];;;;;;;;;;;;;x
+GetBodyQualityResponse_deformed;Even a blind person would notice that the body is damaged. You look at the carver, giving you a smile in return while the guard comes up behind him.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3054]:
```suggestion
GetBodyQualityResponse_deformed;Even a blind person would notice that the body is damaged. You look at the carver, who gives you a smile in return while the guard comes up behind him.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 4]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+GetBodyQualityResponse_excellent;It looks fabulous. These curves, these shapes resemble the statues of ancient masters. It's a perfect vessel for you indeed. You look at the carver, giving you a smile in return.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3055]:
```suggestion
GetBodyQualityResponse_excellent;It looks fabulous. These curves and the shape resembles the statues of ancient masters. It is indeed a perfect vessel for you. You look at the carver, who gives you a smile in return.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 5]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+GetBodyQualityResponse_normal;It looks fine. You remark sharp angles and slight antisymmetry, but it's barely noticeable. It's a good vessel for you indeed. You look at the carver, giving you a smile in return.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3056]:
```suggestion
GetBodyQualityResponse_normal;It looks fine. You spot sharp angles and slight asymmetry, but it's barely noticeable. It is indeed a good vessel for you. You look at the carver, who gives you a smile in return.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 6]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+spiritbinding_soul_transferred_tooltip;Your soul is successfully transfered.;;;;;;;;;;;;;x
+spiritbinding_soul_trapped_tooltip;Your soul is trapped.;;;;;;;;;;;;;x
+EVTOPTA_WCRAC_510;What it's?!;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3059]:
```suggestion
EVTOPTA_WCRAC_510;What went wrong?!;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 7]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+EVTDESC_WCRAC_510;Spiritbinders gather around you muttering their mantra. You feel like you're falling. Trying to find the ground under your feet, you look down and see your body. [Root.GetSpiritbindingOutcome];;;;;;;;;;;;;x
+GetSpiritbindingOutcome_transferred;The soul float toward your new vessel. Finally, flash. With your new eyes, you watch the old empty body falling to its knees.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3061]:
```suggestion
GetSpiritbindingOutcome_transferred;Your soul floats toward your new vessel. Finally, a flash. With your new eyes, you watch the old empty body falling to its knees.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 8]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+GetSpiritbindingOutcome_trapped;You float farther and farther away straight to the firmament. At the last moment of your life, you witness the giant black whirlwind among the clouds.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3062]:
```suggestion
GetSpiritbindingOutcome_trapped;You float farther and farther away, reaching the firmament. At the last moment of your life, you witness the giant black whirlwind among the clouds.;;;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 9]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+build_new_vessel;Build New Vessel;;;;;;;;;;;;;x
+build_new_vessel_desc;Ancient secrets are known to your people. Lifeless rock could be animated, providing a willing or unwilling soul could be captured within. Thus you're able to transfer your soul to a new vessel if the current one doesn't suit you anymore. You will be rejuvenated and lose all the body-related traits.;;;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3064]:
```suggestion
build_new_vessel_desc;Ancient secrets are known to your people. Lifeless rock could be animated, providing a willing or unwilling soul could be captured within. Thus, you're able to transfer your soul to a new vessel if the current one doesn't suit you anymore. You will be rejuvenated and lose all the body-related traits.;;;;;;;;;;;;;x
```

Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv]: Done in 1ab80ba4441f459c577073e118cd7a99e309a891.

File: localisation/00_zuwc_united_localisation.csv [hunk 10]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+EVTDESC_WCRAC_500;You order the most skilled carver you can find to sculpt your new body. Together you career to the capital. The masons who work there assure you that you won't find a better stone in the whole country. They explain to you that there are two types of stone: soft and hard. Your carver notes that it is easier to work soft stone, but a body of hard stone will last longer and possibly be of better quality.;;Ihr beauftragt den versiertesten Skulpteur, den Ihr auftreiben k�nnt, um Euren neuen K�rper zu modellieren. Gemeinsam eilt ihr in die Hauptstadt. Die Maurer, die dort arbeiten, versichern Euch, dass Ihr im ganzen Land keinen besseren Stein finden werdet. Sie erkl�ren Euch, dass es zwei Arten von Gestein gibt: weiches und hartes. Euer Skulpteur merkt an, dass es einfacher ist, weichen Stein zu bearbeiten, aber ein K�rper aus hartem Stein wird l�nger halten und m�glicherweise von besserer Qualit�t sein.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3044]:
```suggestion
EVTDESC_WCRAC_500;You order the most skilled carver you can find to sculpt your new body. Together you journey to the capital. The masons who work there assure you that you won't find a better stone in the whole country. They explain to you that there are two types of stone: soft and hard. Your carver notes that it is easier to work soft stone, but a body of hard stone will last longer and possibly be of better quality.;;Ihr beauftragt den versiertesten Skulpteur, den Ihr auftreiben k�nnt, um Euren neuen K�rper zu modellieren. Gemeinsam eilt ihr in die Hauptstadt. Die Maurer, die dort arbeiten, versichern Euch, dass Ihr im ganzen Land keinen besseren Stein finden werdet. Sie erkl�ren Euch, dass es zwei Arten von Gestein gibt: weiches und hartes. Euer Skulpteur merkt an, dass es einfacher ist, weichen Stein zu bearbeiten, aber ein K�rper aus hartem Stein wird l�nger halten und m�glicherweise von besserer Qualit�t sein.;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv [hunk 11]
```diff
@@ -3041,27 +3041,27 @@ nick_the_unstoppable;the Unstoppable;;;;;;;;;;;;;x
...
+GetSpiritbindingOutcome_trapped;You float farther and farther away, reaching the firmament. At the last moment of your life, you witness the giant black whirlwind among the clouds.;;Ihr schwebt immer weiter weg und erreicht schlie�lich das Himmelszelt. Im letzten Moment Eures Lebens werdet Ihr Zeuge des riesigen schwarzen Wirbelsturms zwischen den Wolken.;;;;;;;;;;;x
+build_new_vessel;Build New Vessel;;Neues Gef�� fertigen lassen;;;;;;;;;;;x
+build_new_vessel_desc;Ancient secrets are known to your people. Lifeless rock could be animated, providing a willing or unwilling soul could be captured within. Thus, you're able to transfer your soul to a new vessel, if the current one doesn't suit you anymore. You will be rejuvenated and lose all the body-related traits.;;Eurem Volk sind uralte Geheimnisse bekannt. Lebloser Fels kann belebt werden, vorausgesetzt, eine bereitwillige oder unwillige Seele kann darin gefangen werden. So w�rt Ihr in der Lage, Eure Seele in ein neues Gef�� zu �bertragen, wenn Euch das jetzige nicht mehr passt. Dadurch w�rdet Ihr verj�ngt und w�rdet alle k�rperbezogenen Eigenschaften verlieren.;;;;;;;;;;;x
```
Comment (LordSilvermane) [inline localisation/00_zuwc_united_localisation.csv:3064]:
```suggestion
build_new_vessel_desc;Ancient secrets are known to your people. Lifeless rock could be animated, providing a willing or unwilling soul could be captured within. Thus, you're able to transfer your soul to a new vessel, if the current one doesn't suit you anymore. You will be rejuvenated and lose all body-related traits.;;Eurem Volk sind uralte Geheimnisse bekannt. Lebloser Fels kann belebt werden, vorausgesetzt, eine bereitwillige oder unwillige Seele kann darin gefangen werden. So w�rt Ihr in der Lage, Eure Seele in ein neues Gef�� zu �bertragen, wenn Euch das jetzige nicht mehr passt. Dadurch w�rdet Ihr verj�ngt und w�rdet alle k�rperbezogenen Eigenschaften verlieren.;;;;;;;;;;;x
```

File: localisation/00_zuwc_united_localisation.csv (hunk 10 above)
Comment (arithon) [reply localisation/00_zuwc_united_localisation.csv:3044]: done

File: localisation/00_zuwc_united_localisation.csv (hunk 11 above)
Comment (arithon) [reply localisation/00_zuwc_united_localisation.csv:3064]: searched the whole text for your change, found it, and when i wanted to comment on it, i read your comment 😅 done

Comment (LordSilvermane) [reply localisation/00_zuwc_united_localisation.csv]: Haha, sorry, it was only when I saw your first done that I realized it didn't highlight the bits I changed.

**Pull request merged**